jobs/ci-kubernetes-e2e-gce-1-6-master-upgrade-cluster.env

```diff
@@ -0,0 +1,12 @@
+### job-env
```

Comment:
according to the [source](https://github.com/kubernetes/kubernetes/blob/b983cc76e15eaa983f90b451829292fdf78ce1b3/test/e2e/upgrades/upgrade.go#L30), I can get that:
```
MasterUpgrade indicates that only the master is being upgraded.
ClusterUpgrade indicates that both master and nodes are being upgraded.
```